README.md

```diff
@@ -14,3 +14,34 @@ part of the array each recursive call considers.
 Analyse the time complexity of your implementation and give a $\Theta$ bound for
 its worst-case runtime. Add your answer, including your reasoning, to this
 markdown file.
+
+### Time complexity and Worst case runtime 
+
+Outer Loop doubles the subarray width, starting at 1 and increases as 2^k until the width goes past n which is the size of the array. 
+
+Total iterations = log<sub>2</sub>n
```

Comment:
The log here doesn't show up anywhere afterwards -- are you sure that your analysis is correct?

Reply:
After review, I think the worst-case is Θ(n^2 log n)